packages/tpetra/core/test/Map/Map_UnitTests.cpp

```diff
@@ -426,7 +426,7 @@ namespace {
     RCP<N2> n2 (new N2);
 
     // create a contiguous uniform distributed map with numLocal entries per node
-    RCP<const Map1> map1 = createUniformContigMapWithNode<LO, GO, N1> (numGlobal, comm, n1);
+    RCP<const Map1> map1 = createUniformContigMapWithNode<LO, GO, N1> (numGlobal, comm);
```

Comment:
@kddevin should a test using the deprecated function signature be kept to ensure it works until the deprecated function signature is actually removed from the code?
We are having a similar discussion for some MueLu deprecation...

Reply:
I don't think we're obligated to keep supporting deprecated functions, but it certainly wouldn't hurt to leave a test in place, as long as it doesn't generate deprecated warnings if deprecated code is disabled.

Reply:
I admit I was not consistent.  In some places, I kept the old interface (e.g., https://github.com/trilinos/Trilinos/blob/e000e39a04969848a2e462def7dbde9e83f822ff/packages/tpetra/core/test/TpetraUtils_MatrixGenerator.hpp#L226 and https://github.com/trilinos/Trilinos/blob/e000e39a04969848a2e462def7dbde9e83f822ff/packages/tpetra/core/test/TpetraUtils_MatrixGenerator.hpp#L176 ).  In other places, I just removed calls to the old interface.